app/settings.py

```diff
@@ -44,11 +44,11 @@
 
 RABBIT_URLS = [RABBIT_URL, RABBIT_URL2]
 
-RABBIT_CORA_QUEUE = os.getenv('SDX_STORE_RABBIT_CORA_QUEUE',
+RABBIT_CORA_QUEUE = os.getenv('SDX_COLLECT_RABBIT_CORA_QUEUE',
```

Comment:
Need to add these env vars to the readme file (and any that have changed/been removed updated)